lib/rules.rb

```diff
@@ -14,19 +17,19 @@ class Rules
     [2, 4, 6]
 ]
 
-  def game_over?(board)
-    winning_combination?(board) || check_for_draw?(board)
+  def game_over?
```

Comment:
Why did you decide to remove the `board` argument? (I see that you moved it to an instance variable, but why?)

Reply:
I felt like my code was littered with passing board arguments. After you asked the question I was wondering if I unnecessarily exposed the board? 🤔 💭